frontend/js/require.config.js

```diff
@@ -35,6 +36,11 @@ require.config({
     waitSeconds: 10,
 
     shim: {
+        "hls": {
+            exports: "hls",
+            deps: ['require']
+            },
```

Comment:
Okay, so I looked into this, and apparently we don't need this section at all. `hls.js` has a UMD and can thus be loaded as is by RequireJS.

Reply:
Now I deleted and works anyway. With this I think the documentation of MEP is not entirely correct.